extensions/rich_text_components/Math/directives/oppia-noninteractive-math.directive.ts

```diff
@@ -36,8 +36,13 @@ angular.module('oppia').directive('oppiaNoninteractiveMath', [
       controller: ['$attrs', function($attrs) {
         var ctrl = this;
         ctrl.$onInit = function() {
-          ctrl.rawLatex = HtmlEscaperService.escapedJsonToObj(
-            $attrs.rawLatexWithValue);
+          ctrl.mathExpressionsContent = HtmlEscaperService.escapedJsonToObj(
```

Comment:
This name sounds better: `mathExpressionsContent` (maybe just Expression (not s))

Reply:
done